src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputLineLengthJsniMethods.java

```diff
@@ -0,0 +1,13 @@
+package com.google.checkstyle.test.chapter4formatting.rule44columnlimit;
+
+public class InputLineLengthJsniMethods {
+  public static native void alertMessage(String msggggggggggggggggggggggggggggggggggggggggggggggggggg) /*-{
+    $wnd.alert(msg);
+    console.log('a really long message here blah blah blah blah blah bruh bruh bruhhhhhhhhhhhhhhhhhhhhhhhhhhhh');
+  }-*/;
+
+  public static native void alertMessage2(String msggggggggggggggggggggggggggggggggggggggggggggggggggg) /*- { // warn
+    $wnd.alert(msg);
+    console.log('a really long message here blah blah blah blah blah bruh bruh bruhhhhhhhhhhhhhhhhhhhhhhhhhhhh'); // warn
+  }-*/;
```

Comment:
This example does not clearly demonstrate what's going on here, IMO. Please do something like:

```suggestion
  
  // JSNI method
  public static native void alertMessage(String msg) /*-{
    $wnd.alert(msg);
    console.log('my very long message .......................................................................................................................................................');
  }-*/;

  // just a comment, no JSNI delimiters
  public static native void alertMessage2(String msg) /*
    $wnd.alert(msg);
    console.log('my very long message .......................................................................................................................................................'); // warn
  */;
```

Reply:
done